samples/bot-conversation/nodejs/resources/UserMentionCardTemplate.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
   "type": "AdaptiveCard",
-  "version": "1.0",
+  "version": "1.5",
   "body": [
```

Comment:
Speak property is missing here:
![image](https://user-images.githubusercontent.com/59052547/209697252-aed58bac-2996-4e20-b03f-d55a06c0866c.png)

Reply:
Added